lib/iris/tests/unit/experimental/ugrid/mesh/test_Connectivity.py

```diff
@@ -61,10 +62,14 @@ def test_indices(self):
 
     def test_read_only(self):
         attributes = ("indices", "cf_role", "start_index", "location_axis")
+        if sys.version_info.minor >= 11:
+            msg = "object has no setter"
+        else:
+            msg = "can't set attribute"
```

Comment:
@rcomer I'd recommend using [pkg_resources.parse_version](https://setuptools.pypa.io/en/latest/pkg_resources.html#utility-functions) from `setuptools` which is [pep440](https://peps.python.org/pep-0440/) compliant :+1: 

So something along the lines of:
```python
from pkg_resources import parse_version
from platform import python_version

msg = "object has no setter" if parse_version(python_version()) >= parse_version("3.11") else "can't set attribute"
```

Reply:
Ah thanks, I wondered if there was a better way to do that 👍

Reply:
`black` turned your one-liner into 5 lines, so I prefer the 4-line loop!